ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,52 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            if (num1 == num2)
```

Comment:
This is good and works.  However, think about what this if-statement is saying: "If this is true, then return true. Otherwise, return false."
Does it make sense that you could just return the conditional?  `return num1 == num2;`